File: C#/Beecrowd/1044 - Multiples.cs
using System;
using System.Linq;

namespace Beecrowd{

    public class Program{
		
    	public static void Main(){
    		
    		int[] scanf = Console.ReadLine().Split().Select(int.Parse).ToArray();
    		
    		if((scanf[1] % scanf[0]) == 0){
				Console.WriteLine("Sao Multiplos");
			}
			else{
				Console.WriteLine("Nao sao Multiplos");
			}
    	}
    }
}

File: JavaScript/Beecrowd/1044 - Multiples.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

// Separa os números em um array
const nums = input.split(' ');

// Converte os números para o tipo numérico
const num1 = Number(nums[0]);
const num2 = Number(nums[1]);

// Verifica se os números são múltiplos
if (num1 % num2 === 0 || num2 % num1 === 0) {
  console.log('Sao Multiplos');
} else {
  console.log('Nao sao Multiplos');
}
